packages/react-native-gesture-handler/src/index.ts

```diff
@@ -162,10 +162,12 @@ export type {
 } from './components/DrawerLayout';
 export { default as DrawerLayout } from './components/DrawerLayout';
 
-export type { NativeDetectorProps } from './v3/NativeDetector/NativeDetector';
-export { NativeDetector } from './v3/NativeDetector/NativeDetector';
+export type { NativeDetectorProps } from './v3/Detectors/common';
+export { NativeDetector } from './v3/Detectors/NativeDetector';
+
+export { LogicDetector } from './v3/Detectors/LogicDetector/LogicDetector';
+export { DelegateDetector } from './v3/Detectors/LogicDetector/DelegateDetector';
```

Comment:
Maybe let's also create `index.ts` in `v3/detectors` (and yes, I'd use lowercase for that directory 😅) which will export these?